index.html

```diff
@@ -35,6 +35,16 @@
         <h1>Helping new talent into the tech world</h1>
       </div>
     </section>
+        <!--Business Partners Section-->
+
+        <section class="partners">
```

Comment:
This `<section>` tag should be on the same indentation level as the previous section, since they are siblings.

Reply:
Hey @otagi thanks for your constructive comments.
I just created the PR cause i was curious to see how it would go(since this is my first contribution in the group project ).
but lately i have tried to make some more amendments (i have added logos ...)
the reason why i put transparencey in some of the CSS classes is because we have some partners' logos in a PNG format.

kindly,Some of your comments are not clear for me(cause i am lucking some basic technical knowledge ) so i need sometime to figure out the mistakes i made and after i will try to solve them immidiately.

Reply:
No worries. It's good that you experiment to see how it works!

Feel free to ask more questions. If something is not clear, it's my fault, and I should improve my explanations. 😉